styles.css

```diff
@@ -9,18 +9,21 @@
 }
 
 body {
-  background: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  background: #de9977;
 }
```

Comment:
I can see that you have put a great deal of effort into this project. There seems to be some overflow on the page. Kindly fix this to enhance the user experience on your website.

![image](https://user-images.githubusercontent.com/20554583/206502838-6101e10a-1473-4a05-9b24-d707a5c8cbda.png)